packages/oui-select/src/templates/match.html

```diff
@@ -15,7 +15,7 @@
         </span>
         <span class="ui-select-match-text"
             ng-hide="$select.isEmpty()"
-            ng-bind-html="$select.selected[$ctrl.match] || $select.selected">
+            ng-bind-html="$ctrl.getPropertyValue($select.selected) || $select.selected">
```

Comment:
Since `getPropertyValue` can return a default value with `get`, maybe we should go with;

```
ng-bind-html="$ctrl.getPropertyValue($select.selected)"
```

 and

```
    getPropertyValue (item) {
        return get(item, this.match, item);
    }
```

What do you think ?